testutil/testnode/full_node.go

```diff
@@ -135,3 +140,64 @@ func DefaultTendermintConfig() *config.Config {
 	tmCfg.Mempool.MaxTxBytes = 22020096 // 21MB
 	return tmCfg
 }
+
+// DefaultGenesisState will create
```

Comment:
```suggestion
// DefaultGenesisState will create a default genesis state, make the `fundedAccounts` validators, and fund them.
```

Reply:
```suggestion
// DefaultGenesisState returns a default genesis state and a keyring with
// accounts that have coins. The keyring accounts are based on the
// fundedAccounts parameter.
```

slightly different doc comment proposal because I don't see how this function makes the `fundedAccounts` validators

Reply:
good catches, ended up going with the second one as they are not validators